### PR TITLE
py-fastdtw: Submission

### PR DESCRIPTION
**Pull request by funasoul**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-fastdtw/Portfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+python.versions         37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append  port:py${python.version}-setuptools \
+                          port:py${python.version}-requests
+
+    depends_lib-append    port:py${python.version}-numpy
+
+    livecheck.type        none
+} else {
```
Comment (herbygillot) [inline python/py-fastdtw/Portfile:32]: The `python` portgroup should automatically be taking care of the `livecheck` details, so you really should only need the `if` part of this, and not the `else`.  Am I mistaken?

File: python/py-fastdtw/Portfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+PortGroup               python 1.0
+
+name                    py-fastdtw
+version                 0.3.4
+categories-append       science
+platforms               darwin
+license                 MIT
+maintainers             nomaintainer
+description             Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity.
+long_description        Python implementation of FastDTW, which is an approximate Dynamic Time Warping (DTW) algorithm that provides optimal or near-optimal alignments with an O(N) time and memory complexity.
```
Comment (herbygillot) [inline python/py-fastdtw/Portfile:13]: Minor nitpick, but I would encourage folding this line using `\`

File: python/py-fastdtw/Portfile (hunk 1 above)
Comment (funasoul) [reply python/py-fastdtw/Portfile]: Thank you for your review.
I have removed the `else` block, squashed the commit, and pushed as 3cf45cd .
Please have a look.

File: python/py-fastdtw/Portfile (hunk 2 above)
Comment (funasoul) [reply python/py-fastdtw/Portfile]: Thank you for your review.
I have added `\` for every 80 characters, squashed the commit, and pushed as 3cf45cd .

**Comment by herbygillot** [comment]
Thank you @funasoul, looks good. Merged.

**Pull request merged**